IntegrationTests/binary.bats

```diff
@@ -15,5 +15,5 @@ teardown() {
     run utica update --platform iOS --valid-simulator-archs "i386 x86_64"
 
     [ "$status" -eq 0 ]
-    [ -d Carthage/Build/iOS/Firebase.framework ]
+    [ -d Carthage/Build/FirebaseAnalytics.xcframework ]
```

Comment:
So, the platform `iOS` gone now? since it's xcframework?

Will it work "backwards" - with older xcodes (and with platform/frameworks) ?

Reply:
UPD: it will work! :) Thanks!